### ISSUE TITLE
[feature request] Global toggle for lsp-ui-doc-enable

### ISSUE THREAD

**Issue by betaprior** [emacs-lsp/lsp-ui#625]
I would like to be able to toggle lsp-ui-doc popups, but using the `lsp-ui-doc-enable` variable I can only seem to control the state on emacs startup. For example, if I start with doc popups enabled but then do `(setq lsp-ui-doc-enable nil)`, it takes no effect.

The best I can do is toggle `lsp-ui-doc-mode`, but that only takes effect for a given buffer.  It would be nice to be able to have global toggles for lsp-ui features that can take effect across all buffers and not just on startup.

**Comment by ericdallo** [comment]
I can't see an easy way to do that, any suggestion @yyoncho?

**Comment by yyoncho** [comment]
> but using the `lsp-ui-doc-enable` variable I can only seem to control the state on emacs startup

I don't think that this is true. After reopening the buffer lsp-ui-doc should not be enabled(at least lsp-mode won't enable it).

**Comment by betaprior** [comment]
> I don't think that this is true. After reopening the buffer lsp-ui-doc should not be enabled(at least lsp-mode won't enable it).

Ah, well I was thinking about more of an on-the-fly toggle that does not involve re-opening buffers. Some of us have problems with never killing buffers ;)
